qiskit_ibm_runtime/options/zne_options.py

```diff
@@ -34,7 +34,52 @@
 
 @primitive_dataclass
 class ZneOptions:
-    """Zero noise extrapolation mitigation options. This is only used by V2 Estimator.
+    """Zero noise extrapolation mitigation options. This is only used by the V2 Estimator.
+
+    ..note::
+
+        Any V2 estimator is guaranteed to return data fields called ``evs`` and ``stds`` that
+        report the desired expectation value estimates and errors, respectively.
+        When ZNE options are enabled in the runtime estimator, additional data is returned.
+
+        In particular, suppose an input pub has observable array shape ``obs_shape`` and parameter
+        values shape ``par_shape``, with corresponding pub shape ``shape``. Then the corresponding
+        pub result will additionally contain:
+
+        1. `pub_result.data.evs_extrapolated` and `pub_result.data.stds_extrapolated`,
+            both with shape ``(*shape, num_extrapolators, num_evaluation_points)``, where
+            ``num_extrapolators`` is the length of the list of
+            ``options.resilience.zne.extrapolators``, and ``num_evaluation_points`` is the length of
+            the list ``options.resilience.extrapolated_noise_factors``. These values provide
+            evaluations of every extrapolator at every specified noise extrapolation value.
+        2. ``pub_result.data.evs_noise_factors``, ``pub_result.data.stds_noise_factors``, and
+           ``ensemble_stds_noise_factors`` all have shape ``(*shape, num_noise_factors)`` where
+           ``num_noise_factors`` is the length of ``options.resilience.zne.noise_factors``. These
+           values provide evaluations of the best-fit model at each of the noise amplifications.
+           In the case of no twirling, both ``*stds*`` arrays will be equal, otherwise,
+           ``stds_noise_factors`` is derived from the spread over twirling samples, whereas
+           ``ensemble_stds_noise_factors`` assumes only shot noise and no drift.
+
+        Metadata is formatted as follows:
+
+        - ``PrimitiveResult.metadata["resilience"]["zne"]``
```

Comment:
I'm always a bit hesitant to specify this kind of information because if in a few weeks we change the metadata in the server (e.g., by adding a field), there is a chance that we may forget to update this docstring

Reply:
We could just remove all of the metadata description.